### PR TITLE
AMG: add command group for linking Azure Monitor workspaces

### PR DESCRIPTION
**Pull request by ABZhang0**
### Related command
``az grafana integrations monitor``

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

### CODE REVIEW

**Review by kairu-ms — COMMENTED** [review]
Why added the whole monitor sdk? Please use the aaz unregistered command for monitor operations.

**Review by kairu-ms — APPROVED** [review]

**Comment by dbradish-microsoft** [comment]
@kairu-ms, where are the examples for these new commands? I expected to see them in help.py.

File: src/amg/azext_amg/_help.py [hunk 1]
```diff
@@ -444,3 +444,28 @@
...
+
+helps['grafana integrations monitor list'] = """
+    type: command
+    short-summary: List all Azure Monitor workspaces linked to a Grafana instance.
+"""
+
+helps['grafana integrations monitor delete'] = """
+    type: command
+    short-summary: Unlink an Azure Monitor workspace from a Grafana instance.
+"""
```
Comment (kairu-ms) [inline src/amg/azext_amg/_help.py:471]: @dbradish-microsoft  are you looking for this?

**Pull request merged**